tests/functional/conftest.py

```diff
@@ -68,3 +69,23 @@ def db_session(db_engine):
         factories.clear_sqlalchemy_session()
         session.close()
         conn.close()
+
+
+@pytest.fixture
+def do_lti_launch(app):
+    def _do_lti_launch(post_params, get_params=None, **kwargs):
+        url = "/lti_launches"
+        if get_params:
+            url += f"?{urlencode(get_params)}"
+
+        return app.post(
+            url,
+            params=post_params,
+            headers={
+                "Accept": "text/html",
+                "Content-Type": "application/x-www-form-urlencoded",
+            },
+            **kwargs,
+        )
+
+    return _do_lti_launch
```

Comment:
```suggestion
    def do_lti_launch(post_params, get_params=None, **kwargs):
        url = "/lti_launches"
        if get_params:
            url += f"?{urlencode(get_params)}"

        return app.post(
            url,
            params=post_params,
            headers={
                "Accept": "text/html",
                "Content-Type": "application/x-www-form-urlencoded",
            },
            **kwargs,
        )

    return do_lti_launch
```

I think our "standard" is to have the inner be the same name as the outer. I couldn't fully tell you why. I think this is to prevent people from "choosing any name they like". I'm not sure what the consequence would be if they did though.